src/orders/components/OrderFulfillStockExceededDialogLine/OrderFulfillStockExceededDialogLine.tsx

```diff
@@ -38,7 +38,7 @@ const OrderFulfillStockExceededDialogLine: React.FC<OrderFulfillStockExceededDia
         thumbnail={line?.thumbnail?.url}
       >
         {line?.productName}
-        {"attributes" in line.variant && (
+        {line.variant && "attributes" in line.variant && (
```

Comment:
looking at the 40'th line, `line` can be undefined, right?

Reply:
It can, but there is an early return guard clause on lines 23-25

Reply:
Is this not `line?.variant?.attributes`?

Reply:
We can't just use `line?.variant?.attributes` because this component works on two types of data, order lines and fulfillment lines. Fulfillment lines don't have attributes, so in order to make typescript happy we need discriminated unions here